File: problems/algorithms/array-prime-numbers/ruby/response/spec/prime_numbers.spec.rb
require_relative '../src/prime_numbers'

context 'Number Tools' do
    it 'looking for the first 4 prime numbers' do
        expect(primes(ed:4)).to eql([2, 3, 5, 7])
    end
    it 'looking for the first 6 prime numbers' do
        expect(primes(ed:6)).to eql([2, 3, 5, 7, 11, 13])
    end
    it 'looking for prime numbers >= 2 and <= 10' do
        expect(primes(bg:2, ed:10)).to eql([2, 3, 5, 7])
    end
    it 'looking for prime numbers >= 2 and <= 20' do
        expect(primes(bg:2, ed:20)).to eql([2, 3, 5, 7, 11, 13, 17, 19])
    end
    it 'looking for prime numbers >= 10 and <= 2' do
        expect(primes(bg:10, ed:2)).to eql([])
    end
    
end

File: src/pages/exercises/array-prime-numbers/_codes/php/response/spec/prime-numbers.spec.php
<?php
require __DIR__ . '/../src/prime-numbers.php';

describe('Number Tools', function () {
  it('looking for the first 4 prime numbers', function () {
    expect(primes(4))->toEqual([2, 3, 5, 7]);
  });

  it('looking for the first 6 prime numbers', function () {
    expect(primes(6))->toEqual([2, 3, 5, 7, 11, 13]);
  });

  it('looking for prime numbers >= 2 and <= 10', function () {
    expect(primes(2, 10))->toEqual([2, 3, 5, 7]);
  });

  it('looking for prime numbers >= 2 and <= 20', function () {
    expect(primes(2, 20))->toEqual([2, 3, 5, 7, 11, 13, 17, 19]);
  });

  it('looking for prime numbers >= 10 and <= 2', function () {
    expect(primes(10, 2))->toEqual([]);
  });
});
